### ISSUE TITLE
[Bug] Tasks plugin not cover all function when Lazy plugin loader enable

### ISSUE THREAD

**Issue by l2eused** [alangrainger/obsidian-lazy-plugins#8]
In normally, with **Tasks** plugin and **Kanban** plugin.

When tick checkbox or move card to completed list in kanban board. Tasks will add completed date (if enable) to a checkbox/card.

When **Lazy plugin loader** plugin enable, that feature is missing even I set **Tasks** and **Kanban** plugin as **instant** load.

This didn't occer in previous version. Maybe version that when enable plugin loader debugging, it will show all plugins that enable plus *lazy plugin*. (In this version it only shows *lazy plugin* not instant load plugin)

**Comment by alangrainger** [comment]
Thanks @l2eused - I can replicate the issue. To resolve it I have reverted the change [suggested by Obsidian team here](https://github.com/obsidianmd/obsidian-releases/pull/3998#issuecomment-2318012016).

If you upgrade to 1.0.13 and set Kanban and Tasks to Instant, it will now work correctly.

(For future reference, I can confirm the loading the plugins with any other method causes the completed date to not function. The order of loading the plugins doesn't matter either.)

**Comment by alangrainger** [comment]
You'll need to restart a couple of times for the new settings to take effect before you can see the final result of the update.

**Comment by alangrainger** [comment]
This is what I get, which is what I would expect:

![image](https://github.com/user-attachments/assets/2ffb87fd-e1f7-4c93-8a0c-20b30fc2dd56)

Just make sure you restart a couple of times to lock in the individual plugin startup settings after the 1.0.13 update.

**Comment by alangrainger** [comment]
You are right, I can see there is something strange about the combination of those two plugins. The kanban plug-in does indeed appear to the load instantly, even if you haven't sent to be a delay. I will keep diagnosing this.

**Comment by alangrainger** [comment]
Should be fixed now in 1.0.15. It was a missing setting being `undefined` and causing all plugins to instantly start.
